src/sentry/api/serializers/models/group.py

```diff
@@ -811,16 +811,25 @@ def __seen_stats_impl(
 
         project_id = issue_list[0].project_id
         item_ids = [g.id for g in issue_list]
+        tenant_ids = {"organization_id": issue_list[0].project.organization_id}
```

Comment:
Just noting that we preload group projects here https://github.com/getsentry/sentry/blob/c8bbb69c609b5537d03f3da8c0aafda338e89293/src/sentry/api/serializers/models/group.py#L214, so this won't cause an extra query.

Reply:
I'm new to Django, are you saying I'm causing an extra query by doing `.project.organization_id`? How can I avoid this if so?

Reply:
No, just noting for anyone else reading that it won't cause any extra load